### DISCUSSION TITLE
Linked Data + email plus addressing?

### DISCUSSION THREAD

**Discussion by outlaw-dame** [assemblee-virtuelle/activitypods#171] [General]
I have included screenshots for context. I am wondering if it would make sense to solve for the attached issues by ActivityPods supporting email's plus addressing + Linked Data. Obviously, open to feedback and better alternatives. I think it would benefit users depending on the kind of accounts and purposes. For the example references below they could simply use one Flipboard account for Semafor but with plus addressing they can still fulfil their needs

<img width="710" alt="Screenshot 2024-02-15 at 11 20 00" src="https://github.com/assemblee-virtuelle/activitypods/assets/141657791/e8dcae28-dd37-4d67-9177-0014d5d2fcd6">
<img width="631" alt="Screenshot 2024-02-15 at 11 20 16" src="https://github.com/assemblee-virtuelle/activitypods/assets/141657791/b568ad08-0603-4e7a-80b1-3e3e414701fc">
<img width="302" alt="Screenshot 2024-02-15 at 11 20 34" src="https://github.com/assemblee-virtuelle/activitypods/assets/141657791/a67f0c7f-669e-4a0c-b704-ca6d8433d5b7">

**Comment by Laurin-W** [comment]
Hey Damon, I'm no quite sure yet how this would look like and how this could be implemented / used, standarized and so on.
But if I understand correctly, this is about handling multiple (sub)identities with the same account. I had a chat with a person a while ago and they referred to an article (which they were gonna find and which I planning to read) about this topic.

I'll get back here once I have a better conception of the topic and I'll link the article here.

What kinds of use cases do you envision for your own application for this feature?

**Comment by outlaw-dame** [comment]
The kinds of use cases I envision are Enhanced Identity Verification: With Linked Data, users could connect their profile to other verified online identities (website, social accounts). This network of linked data points strengthens a user's online presence. This verifiable linked data would make it easier to distinguish legitimate accounts from potentially fake or bot accounts.
Simplified Cross-Instance Migration: A Linked Data profile can act as a 'digital passport.' Migrating to a new instance could be as streamlined as using the same identity credentials on the new server, instantly pulling associated profile data
Linked Data can also be used to create what I call "dynamic tags" Semantic Hashtags transcend traditional tagging by embedding linked data, transforming them into dynamic, context-aware tools that enhance content discoverability and engagement. Each hashtag is linked to a URI that provides detailed descriptions, relationships to other concepts, and even synonyms, making them far more informative. They bridge content across diverse platforms through their interconnected nature, enabling users to explore not only related posts but also broader concepts and discussions. Furthermore, Semantic Hashtags dynamically aggregate content from multiple sources, offering live feeds on specific topics and enabling advanced discovery and analysis through rich metadata, such as sentiment tracking and concept filtering.
Use cases would be enhanced content discovery, even tracking & participation (SuperBowl2024), personalised content curation and recommendations.
